.rubocop.yml

```diff
@@ -12,7 +12,7 @@ AllCops:
     - 'vendor/**/*'
     - 'config/environments/*.rb'
     - 'bin/*'
-
+    - 'app/controllers/application_controller.rb'
```

Comment:
You should avoid to disable all cops for a specific file.
Prefer ignoring cops on certain parts of your file, or at least only disabling specific cops.